### PR TITLE
Branch for the front page

### PR DESCRIPTION
**Pull request by Zelalem1222**
This is a mobile-size portfolio designed with HTML and CSS. This portfolio also has HTML and CSS linter.  also, It includes a link to different social media.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav>
+      
+            <h3>My Logo</h3>
+            <i class="fa-solid fa-bars fa-lg"></i>
+        
+    </nav>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Kindly ensure that the navbar is enclosed inside a `header` tag for semantic purposes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<script src="https://kit.fontawesome.com/f7e5caa7f7.js" crossorigin="anonymous"></script>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">   
+<link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav>
+      
+            <h3>My Logo</h3>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly make sure that the logo is wrapped inside an anchor `a` tag instead of the `h3`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            <i class="fa-solid fa-bars fa-lg"></i>
+        
+    </nav>
+    <header >
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly use a `section tag here instead of `header` tag for semantic purposes

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <h4>LET’S CONNECT</h4> 
+              <span class="vector">
+                <i class="fa-brands fa-twitter fa-xl"></i>
+                <i class="fa-brands fa-linkedin-in fa-xl"></i>
+                <img class="vector5" src="images/vector5.svg" alt="">
+                <img class="vector6" src="images/vector6.svg" alt="">
+                <img class="vector7" src="images/vector7.svg" alt="">
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)
- [ ] Please kindly use the icons as specified in the design. You can download them by exporting

File: index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        <img class="rectangle4" src="images/Rectangle4.svg" alt="">
+        <img class="ellipse8" src="images/Ellipse8.svg" alt="">
+        <img class="ellipse5" src="images/Ellipse5.svg" alt="">
+        <img class="rectangle7" src="images/Rectangle7.svg" alt="">
+        <img class="ellipse12" src="images/Ellipse12.svg" alt="">
+        <img class="ellipse13" src="images/Ellipse13.jpg" alt="">
+        <img class="ellipse15" src="images/Ellipse15.svg" alt="">
+        <img class="ellipse4" src="images/Ellipse4.svg" alt="">
+        <img class="rectangle5" src="images/Rectangle5.svg" alt="">
+    </div>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please ensure to include background images by using the `background-img` CSS property. Kindly remove these `img` tags and include the background image from CSS

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+                Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <h4>LET’S CONNECT</h4> 
+              <span class="vector">
+                <i class="fa-brands fa-twitter fa-xl"></i>
+                <i class="fa-brands fa-linkedin-in fa-xl"></i>
+                <img class="vector5" src="images/vector5.svg" alt="">
+                <img class="vector6" src="images/vector6.svg" alt="">
```
Comment (AnselemOdims) [inline index.html]: - [ ] At the moment, there is horizontal scroll which does not give a good user experience. Please kindly remove it

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Zelalem1222 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please make your pull request summary more descriptive
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+        <img class="ellipse5" src="images/Ellipse5.svg" alt="">
+        <img class="rectangle7" src="images/Rectangle7.svg" alt="">
+        <img class="ellipse12" src="images/Ellipse12.svg" alt="">
+        <img class="ellipse13" src="images/Ellipse13.jpg" alt="">
+        <img class="ellipse15" src="images/Ellipse15.svg" alt="">
+        <img class="ellipse4" src="images/Ellipse4.svg" alt="">
+        <img class="rectangle5" src="images/Rectangle5.svg" alt="">
+        <img class="ellipse11" src="images/Ellipse11.png" alt="">
+        <img class="rectangle6" src="images/Rectangle6.png" alt="">
+    </div>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly as asked by the previous reviewer, Please ensure to include background images by using the `background-img` CSS property. You can download all these images in one image from the Figma design and set it using `background-img` CSS property.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Zelalem1222, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Zelalem1222,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**